### PR TITLE
SaveToCloudButton: Add Cypress test

### PR DESCRIPTION
**Pull request by jespicas**
This PR contains 3 Test, on test per each Cloud button.
Each test after click on the button check if shown a popup with corresponding text.
And click on button done, to return homepage.
Screenshot from 2020-10-07 21-41-16

### CODE REVIEW

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+context('Click SaveToCloud buttons', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should toggle GoogleDrive SaveToCloudButton', () => {
+    cy.get('#root > div > aside > div > div:nth-child(3) > div:nth-child(2) > button.ui.yellow.big.icon.button').click();
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.content > div > div > p').contains('Google Drive')
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.actions > button').click();
```
Comment (aamay001) [inline cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js:12]: You probably want to remove this last click from the test.

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  });
+
+  it('should toggle DropBox SaveToCloudButton', () => {
+    cy.get('#root > div > aside > div > div:nth-child(3) > div:nth-child(2) > button.ui.twitter.big.icon.button').click();
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.content > div > div > p').contains('Dropbox')
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.actions > button').click();
```
Comment (aamay001) [inline cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js:18]: You probably want to remove this last click from the test.

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  });
+
+  it('should toggle OneDrive SaveToCloudButton', () => {
+    cy.get('#root > div > aside > div > div:nth-child(3) > div:nth-child(2) > button.ui.facebook.big.icon.button').click();
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.content > div > div > p').contains('OneDrive')
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.actions > button').click();
```
Comment (aamay001) [inline cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js:24]: You probably want to remove this last click from the test.

**Review by aamay001 — CHANGES_REQUESTED** [review]
The SaveToCloudButton tests are failing. Please try my suggestions below.

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js (hunk 1 above)
Comment (jespicas) [reply cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js]: Updated

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js (hunk 2 above)
Comment (jespicas) [reply cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js]: Updated

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js (hunk 3 above)
Comment (jespicas) [reply cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js]: Updated

**Comment by jespicas** [comment]
Somthing it's missing in pipeline right!?

**Pull request merged**